packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3231,6 +3300,59 @@ export default class Meeting extends StatelessWebexPlugin {
       });
   }
 
+  /**
+   * Enable dial in for audio
+   * @param {String} resourceId
+   * @returns {Promise} once the move has been completed
+   * @public
+   * @memberof Meeting
+   */
+  dialInPstn() {
```

Comment:
I think if we were to use "usePhoneAudio()" we should take in a param to specify whether it's dial in or dial out. Otherwise, we should have separate methods for dialing in and dialing out. Which would work best?

Reply:
i like one function rather then two , we can keep the one you created as private function and have a usePhoneAudio as public

Reply:
If we want to use one method for both, then add an optional parameter, `phoneNumber` that will be used for dial-out. for now, if `phoneNumber` is passed, send back an error saying that calling a phone number is not supported yet